packages/gafl-webapp-service/src/pages/multibuy/view-licences/__tests__/route.spec.js

```diff
@@ -160,10 +211,28 @@ describe('view licences > getData', () => {
       const returnValue = Symbol('return value')
       displayStartTime.mockReturnValueOnce(returnValue)
 
-      const result = await getData(getSampleRequest())
+      const result = await getData(getSampleRequest({ currentPermission: getMockPermission() }))
```

Comment:
`getSampleRequest` defaults the value of `currentPermission` to `getMockPermission()`, so this is unnecessary.